src/app/pages/common/entity/entity-form/entity-form.component.ts

```diff
@@ -533,6 +539,12 @@ export class EntityFormComponent implements OnInit, OnDestroy, OnChanges, AfterV
                         (res) => {
                           this.loader.close();
                           this.loaderOpen = false;
+
+                          if (res.error) {
```

Comment:
wonder if we need

`(res.editCallJob && res.error)`

?

Reply:
Thanks for your pointing out.